api/v1beta2/flowcollector_types.go

```diff
@@ -635,37 +591,10 @@ type FlowCollectorLoki struct {
 	// Set `enable` to `true` to store flows in Loki. It is required for the OpenShift Console plugin installation.
 	Enable *bool `json:"enable,omitempty"`
 
-	//+kubebuilder:default:="1s"
```

Comment:
Like for the ebpf agent, I think batchWait and batchSize could be good to keep not in Debug, for performance tuning?
Especially BatchSize is something users need to keep consistent with the msg max size defined on loki server-side.

Reply:
Well I based that on the Jira you initially wrote :thinking: 
If you changed your mind I can revert. Just let me know.

Reply:
On my side I would feel better as a user to have that elsewhere but `debug` is a strong word here. 
Maybe we could just concider renaming the `debug` sections to something more generic such as `advanced` ?